README.md

```diff
@@ -60,18 +60,14 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Awesome books] <a name="about-project"></a>
 
-> This project is build a basic website that allows users to add/remove books from a list
-
-**[your_project__name]** is a...
+**[Awesome books]** is a...
```

Comment:
- 🙏Please add a description of your project here. This will enable others👨‍👩‍👧‍👦 to understand what the project is about.